dot/rpc/modules/childstate.go

```diff
@@ -68,3 +75,26 @@ func (cs *ChildStateModule) GetKeys(_ *http.Request, req *GetKeysRequest, res *[
 	*res = hexKeys
 	return nil
 }
+
+// GetStorageHash returns the hash of a child storage entry
+func (cs *ChildStateModule) GetStorageHash(_ *http.Request, req *GetStorageHash, res *string) error {
+	if req.Hash == common.EmptyHash {
+		req.Hash = cs.blockAPI.BestBlockHash()
+	}
+
+	stateRoot, err := cs.storageAPI.GetStateRootFromBlock(&req.Hash)
+	if err != nil {
+		return err
+	}
+
+	item, err := cs.storageAPI.GetStorageFromChild(stateRoot, req.KeyChild, req.EntryKey)
+	if err != nil {
+		return err
+	}
+
+	if len(item) > 0 {
+		*res = common.BytesToHash(item).String()
+	}
+
```

Comment:
the return value should also be an optional, but this will return the same value for `item == nil` and an item with length zero that isn't nil

Reply:
Done!